### ISSUE TITLE
Consider making the accessible name optional for the table role

### ISSUE THREAD

**Issue by mfairchild365** [w3c/aria#1039]
This would better match the HTML standard, where it is optional. ARIA 1.1 states that the accessible name is required and sourced from the author.

**Comment by johnfoliot** [comment]
**WCAG 2.0/2.1 - Success Criterion 1.3.1 Info and Relationships (Level A)**
Information, structure, and relationships conveyed through presentation can be programmatically determined or are available in text.

And:
[https://www.w3.org/WAI/WCAG21/Techniques/html/H39](https://www.w3.org/WAI/WCAG21/Techniques/html/H39)

So while the HTML spec is silent here, we have other Recommendations that suggest a table MUST have an accessible name (inferred via WCAG 2.0 SC 1.3.1, suggested via Technique H39), and loads of Best Practice recommendations. 

I support the current ARIA 1.1 requirement as authored (ARIA 1.1 states that the accessible name is required and sourced from the author.)

**Comment by johnfoliot** [comment]
...I'll also add that one of the intentions of Silver is to address current 'gaps' like this one, and ensure that content blocks (like tables) will explicitly require an accessible name going forward.

**Comment by mfairchild365** [comment]
That's a really good point. Thank you, @johnfoliot. A name for tables is commonly referred to as a 'best practice' rather than an actual WCAG failure, hence this request. Your perspective has convinced me that it is a requirement.  I rescind my request.
